code/modules/client/preferences.dm

```diff
@@ -275,8 +278,10 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 			dat += "<br><br>"
 
 			dat += "<b>Custom Job Preferences:</b><BR>"
-			dat += "<a href='?_src_=prefs;preference=ai_core_icon;task=input'><b>Preferred AI Core Display:</b> [preferred_ai_core_display]</a><br>"
 			dat += "<a href='?_src_=prefs;preference=agent_dept;task=input'><b>Preferred Agent Department:</b> [prefered_agent_department]</a><BR>"
+			dat += "<a href='?_src_=prefs;preference=seph_dept;task=input'><b>Preferred Sephirah Department:</b> [prefered_sephirah_department]</a><BR>"
+			dat += "<a href='?_src_=prefs;preference=seph_body;task=input'><b>Preferred Sephirah Body:</b> [prefered_sephirah_bodytype]</a><BR>"
+			dat += "<a href='?_src_=prefs;preference=seph_boxcolor;task=input'><b>Preferred Sephirah Box Color:</b> [prefered_sephirah_boxcolor]</a><BR>"
```

Comment:
```suggestion
			dat += "<b>Preferred Sephirah Box Color:</b> <span style='border: 1px solid #161616; background-color: #[prefered_sephirah_boxcolor];'>&nbsp;&nbsp;&nbsp;</span> <a href='?_src_=prefs;preference=seph_boxcolor;task=input'>Change</a><BR>"
```
![image](https://github.com/vlggms/lobotomy-corp13/assets/110272328/83013298-b1e8-444b-9c98-ecc3d1bd812b)
This way you can actually see the color you're using.

Reply:
Thanks, UI really is my weak point.